src/node/rpc/common_endpoint_registry.h

```diff
@@ -195,6 +195,29 @@ namespace ccf
         .set_auto_schema<void, GetNetworkInfo::Out>()
         .install();
 
+      auto get_nodes_by_rpc_address = [this](auto& args, nlohmann::json&& params) {
```

Comment:
```suggestion
      auto get_nodes_by_rpc_address = [this](
                                        auto& args, nlohmann::json&& params) {
```
Minor formatting fix.